tests/DocumentationExamplesTest.php

```diff
@@ -1821,9 +1821,7 @@ public function testQueryableEncryption(): void
             $this->markTestSkipped('Explicit encryption tests require MongoDB 7.0 or later');
         }
 
-        if (! $this->isEnterprise()) {
```

Comment:
In [PHPLIB-892](https://jira.mongodb.org/browse/PHPLIB-892?focusedCommentId=5447736&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-5447736) you wrote:

> I'll run the `DocumentationExamplesTest::testQueryableEncryption()` with `crypt_shared` and/or `mongocryptd` available with the community server. If it passes, let's remove the `isEnterprise()` method and change the logic in `DocumentationExamplesTest::testQueryableEncryption()` to instead check if either `crypt_shared` or `mongocryptd` is available.

`skipIfClientSideEncryptionIsNotSupported()` doesn't check for either of those things. Did something change since you wrote that comment?

Also, in an earlier comment on that issue I pointed out that ``DocumentationExamplesTest::testQueryableEncryption()`` did require `crypt_shared` and/or `mongocryptd` (since it uses queryable encryption) but the prose test did not (since it explicitly encrypts). You could run the prose tests w/o `crypt_shared` and `mongocryptd` to verify that.

Reply:
@GromNaN: In light of https://github.com/mongodb/specifications/pull/1439, I would be in favor of adding a check for `crypt_shared` and/or `mongocryptd` to the base `skipIfClientSideEncryptionIsNotSupported()` method.

A call to `skipIfClientSideEncryptionIsNotSupported()` could then be used here (after the topology and 7.0+ version check). I'm aware the version check in `skipIfClientSideEncryptionIsNotSupported()` would be redundant, but that's OK.

You could then also remove the explicit crypt_shared/mongocryptd check in `ClientSideEncryptionSpecTest::setUp()`.

With that done, `skipIfClientSideEncryptionIsNotSupported()` would become consistent with `UnifiedTestRunner::isClientSideEncryptionSupported()` and all of our tests would perform the same checks.

Reply:
> I'm aware the version check in skipIfClientSideEncryptionIsNotSupported() would be redundant, but that's OK.

There is a slight difference between the server version check and the feature compatibility version check in `skipIfClientSideEncryptionIsNotSupported`. So this is not redundant.

Reply:
I followed your instructions.

Reply:
Aye, technically different checks. I said "redundant" as our test suite is never run on mixed-version or upgraded replica sets, so FCV will always equate the server version.